modules/auxiliary/scanner/http/docker_enum.rb

```diff
@@ -0,0 +1,49 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name'        => 'Docker Server Version Scanner',
+      'Description' => 'This module attempts identify the version of the Docker Server running.',
```

Comment:
To identify? Of a? Running on a host?

Reply:
Initially I´m just giving the version of the Docker Server.

Anyhow I could also give other useful information like OS, Architecture, etc.
Do you think it´s worth to add more info?

Note: If you enable verbose mode you see everything.

Reply:
To be more clear this is the info it gives:
```
"Version"=>"18.03.1-ce" (I report this!)
"ApiVersion"=>"1.37"
"Arch"=>"amd64"
"BuildTime"=>"2018-04-26T07:15:30.000000000+00:00"
"GoVersion"=>"go1.9.5"
"KernelVersion"=>"4.13.0-45-generic"
"MinAPIVersion"=>"1.12"
"Os"=>"linux"
```

Reply:
Sorry, I was addressing the wording, hah.

I was thinking something like this: `This module attempts to identify the version of a Docker Server running on a host.`.

Reply:
I'd actually add a note about `VERBOSE` printing all information, too. Fetch as much info as is feasible!

Reply:
Fixed in 7b1e7eb085